chain/get_ancestors.go

```diff
@@ -51,28 +51,12 @@ func GetRecentAncestors(ctx context.Context, base types.TipSet, provider TipSetP
 		earliestAncestorHeight = types.NewBlockHeight(uint64(0))
 	}
 
-	// Step 1 -- gather all tipsets with a height greater than the earliest
```

Comment:
Please keep this comment for readability and update it to be accurate: "gather all tipsets up to the first tipset with a height less than or equal to the earliest possible proving period start"